app/team.go

```diff
@@ -160,6 +161,101 @@ func (a *App) SoftDeleteAllTeamsExcept(teamID string) *model.AppError {
 	return nil
 }
 
+func shouldCreateOnboardingLinkedBoard(a *App, c request.CTX, teamId string) bool {
+	ffEnabled := a.Config().FeatureFlags.OnboardingAutoShowLinkedBoard
+	if !ffEnabled {
+		return false
+	}
+
```

Comment:
you can check here that board is enabled as a product because you cannot proceed without it

Reply:
You could use `a.HasBoardProduct()` to check this